gsa/src/web/pages/performance/performancepage.js

```diff
@@ -376,10 +369,8 @@ const mapDispatchToProps = (dispatch, {gmp}) => {
 
   if (gmp.settings.enableGreenboneSensor) {
```

Comment:
I've taken a look at the code again to be sure we are doing the correct thing with the filter here. IMHO you can just remove the if here and below to always use `Filter.fromString('type=' + GREENBONE_SENSOR_SCANNER_TYPE)`
Maybe also rename the `SLAVE_SCANNER_FILTER` to `SENSOR_SCANNER_FILTER`